### PR TITLE
More small fixes

### PR DESCRIPTION
**Pull request by MrTimscampi**
* Prevent Axe from running manually
  This requires you to run it manually in the console, but fixes the console refresh that swallows errors
* Corrects the way parent item IDs are found in the home header, fixing the undefined error in some cases
* Mirror jf-web's sorting for albums on the artist pages
  With jellyfin/jellyfin-web#2265, sorting for the albums is done properly. We were already filtering by PremiereDate, but this adds ProductionYear and SortName as fallbacks, to mimic jf-web.
* Use thumb images for thumb-size cards (Except libraries, folders and user views)
  Ideally, imageUtils would naturally fallback to the Backdrop if there is no thumb, but that's for after ferfega's refactoring effort.

### CODE REVIEW

**Review by ferferga — APPROVED** [review]

File: components/Item/Card.vue [hunk 1]
```diff
@@ -261,8 +261,13 @@ export default Vue.extend({
...
     getImageType(): ImageType {
-      if (this.shape === 'thumb-card' && this.item.Type === 'Movie') {
-        return ImageType.Backdrop;
+      if (
+        this.shape === 'thumb-card' &&
+        !['CollectionFolder', 'Folder', 'UserView'].includes(
+          this.item?.Type || ''
+        )
+      ) {
+        return ImageType.Thumb;
```
Comment (Camc314) [inline components/Item/Card.vue:270]: Why is this changed?

Comment (MrTimscampi) [reply components/Item/Card.vue:270]: Because that's what jf-web does and it's what makes the most sense. We need #520 to be merged in order to implement proper fallback for these, but this is how it should've worked from the start.

Comment (Camc314) [reply components/Item/Card.vue:270]: This breaks all card images in next-up and resume. Surely we shouldn't merge it unless the fix/refactor is implemented as well

Comment (MrTimscampi) [reply components/Item/Card.vue:270]: Temporarily fixed in #526
